question/bank/managecategories/category.php

```diff
@@ -93,7 +96,7 @@
 
 if ($param->delete) {
     if (!$category = $DB->get_record("question_categories", array("id" => $param->delete))) {
-        print_error('nocate', 'question', $thispageurl->out(), $param->delete);
+        throw new moodle_exception('nocate', 'question', $thispageurl->out(), $param->delete);
     }
 
     question_remove_stale_questions_from_category($param->delete);
```

Comment:
looks like this method is exclusive to category feature, i think it can be moved to the plugin, also if moved, the questionlib test need to point to the class in the plugin, please have a look.

Reply:
done, has been moved.